docs/user-guide/creating-application/workflow/cd-pipeline.md

```diff
@@ -3,20 +3,39 @@ Once you are done creating your CI pipeline, you can move start building your CD
 
 ## Creating CD Pipeline
 
-![](../../../.gitbook/assets/cd-pipeline-console.png)
+![](../../../.gitbook/assets/ca-workflow-cd-add.png)
 
-Click on **“+”** sign on CI Pipeline to attach a CD Pipeline to it.
+Click on **“+”** sign on CI Pipeline to attach a CD Pipeline to it. A basic `Create deployment modal` will pop up.
 
-![](../../../.gitbook/assets/create-cd-pipeline%20%283%29%20%283%29.png)
+![](../../../.gitbook/assets/ca-workflow-basic.png)
 
-One can have a single CD pipeline or multiple CD pipelines connected to the same CI Pipeline. Each CD pipeline corresponds to only one environment, or in other words, any single environment of an application can have only one CD pipeline.
+This modal is divided into two sections
 
-So, the images created by the CI pipeline can be deployed into multiple environments through different CD pipeline originating from a single CI pipeline.
+* **Select Environment**
+* **Deployment Strategy**
 
-If you already have one CD pipeline and want to add more, you can add them by clicking on the `+` sign and then choosing the environment in which you want to deploy your application. Once a new CD Pipeline is created for the environment of your choosing, you can move ahead and configure the CD pipeline as required.
+### 1. Select Environment
+This section further including two inputs:
+
+**\(a\) Deploy to Environment**
+
+Select the environment where you want to deploy your application. Once you select the environment, it will display the `Namespace` corresponding to your selected environment automatically.
 
+**\(b\) Namespace**
+
+As we discussed earlier, Selected environment in `Deploy to Environment` will display here automatically.
```

Comment:
As discussed earlier, this field will be automatically populated with the `Namespace` corresponding to the `Environment` selected in previous step.